design/FY2023/NFP-Space Sizing and HVAC.md

```diff
@@ -114,48 +138,198 @@ SpaceHVAC:EquipmentConnections,
+* *Would likely also add a ZoneHVAC:SpaceDistributionList that's referenced at the end of the ZoneHVAC:EquuipmentConnections object
+```
+ZoneHVAC:SpaceDistribution:NonAirFlow,
+       \extensible:2
+       \memo Distributes the output from a piece of zone equipment to one or more Spaces in the Zone.
+   A1, \field Name
+       \required-field
+       \reference SpaceDistribution
+   A2, \field Zone Equipment Object Type
+       \required-field
+       \type choice
+       \key ZoneHVAC:Baseboard:RadiantConvective:Electric
+       \key ZoneHVAC:Baseboard:RadiantConvective:Water
+       \key ZoneHVAC:Baseboard:RadiantConvective:Steam
+       \key ZoneHVAC:CoolingPanel:RadiantConvective:Water
+       \key ZoneHVAC:Baseboard:Convective:Electric
+       \key ZoneHVAC:Baseboard:Convective:Water
+       \key ZoneHVAC:HighTemperatureRadiant
+       \key ZoneHVAC:LowTemperatureRadiant:VariableFlow
+       \key ZoneHVAC:LowTemperatureRadiant:ConstantFlow
+       \key ZoneHVAC:LowTemperatureRadiant:Electric
+   A3, \field Zone Equipment Name
+       \required-field
+       \type object-list
+       \object-list ZoneEquipmentNames
+   A5, \field Thermostat Control Method
+       \type choice
+       \key SingleSpace
+       \key SpaceAverage
+       \default SpaceAverage
+   A6, \field Thermostat Space Name
+       \note Used with SingleSpace thermostat control method
+       \type object-list
+       \object-list SpaceNames
+   A7, \field Space Sizing Basis
+       \type choice
+       \default DesignLoad
+       \key DesignLoad
+       \key FloorArea
+       \key Volume
+       \key PerimeterLength
+   A8, \field Space 1 Name
+       \begin-extensible
+       \required-field
+       \type object-list
+       \object-list SpaceNames
+   N1, \field Space 1 Output Fraction
+       \default autosize
        \units dimensionless
-       \type real
-       \minimum 0
+       \minimum 0.0
        \maximum 1.0
        \autosizable
+   A9, \field Space 2 Name
+       \required-field
+       \type object-list
+       \object-list SpaceNames
+   N2, \field Space 2 Output Fraction
        \default autosize
+       \units dimensionless
+       \minimum 0.0
+       \maximum 1.0
+       \autosizable
 ```
 
-### ZoneHVAC:AirDistributionUnit
-* *Add new field.*
+### SpaceHVAC:EquipmentConnections
+* *New object to connect nodes to a Space, same as ZoneHVAC:EquipmentConnections, but for Spaces.*
 
-  A6 ; \field Space Splitter Name
+```
+SpaceHVAC:EquipmentConnections,
+       \memo Specifies the HVAC equipment connections for a Space. Node names are specified for the
+       \memo Space air node and air inlet nodes.
+  A1 , \field Space Name
+       \required-field
        \type object-list
-       \object-list SpaceSplitters
+       \object-list SpaceNames
+  A2 , \field Space Air Node Name
+       \required-field
+       \type node
+  A3 ; \field Space Air Inlet Node or NodeList Name
+       \type node
```

Comment:
I did not understand this on the first go around. Why doesn't a space have an exhaust or return outlet node? Will these nodes be implied as connecting to a zone exhaust/return outlet? and how to know which space has which outlet type? If an air loop connects to these spaces then yes, the air flow would sum to a zone return node. If a PTAC is connected to a space then the space would need an exhaust node. Also need a space exhaust node for "space" exhaust fans.

Reply:
Yeah, I was hoping to avoid space-level return/exhaust nodes, but that might not work out.